### PR TITLE
[update] fastem: blank the beam when the stage moves during acquisitions

### PR DESCRIPTION
**Pull request by tepals**

### CODE REVIEW

**Comment by tepals** [comment]
see SDEV-1163

File: src/odemis/acq/fastem.py [hunk 1]
```diff
@@ -658,6 +659,7 @@ def acquire_roa(self, dataflow):
             # Update the time left for the acquisition.
             expected_time = len(self._fields_remaining) * total_field_time
             self._future.set_progress(start=time.time(), end=time.time() + expected_time)
+            self._scanner.blanker.value = True  # blank the beam
```
Comment (nandishjpatel) [inline src/odemis/acq/fastem.py:662]: Line 662 can go to the line above 639?

Comment (tepals) [reply src/odemis/acq/fastem.py:662]: good point, that also works

Comment (pieleric) [reply src/odemis/acq/fastem.py:662]: If use use the blanker, do it fully, and only unblank while acquiring. So I'd say move this line just after line 650.

**Review by pieleric — APPROVED** [review]

**Review by nandishjpatel — APPROVED** [review]

**Pull request merged**